frontend/src/components/common/icon/icon.tsx

```diff
@@ -11,53 +11,91 @@ import { ReactComponent as Signout } from '../../../assets/img/signout.svg';
-      return <Compas className={getColor(color)} width={`${size}px`} height={`${size}px`} />;
+      return <Compas className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
 
     case IconName.FOLLOW:
-      return <Follow className={getColor(color)} width={`${size}px`} height={`${size}px`} />;
+      return <Follow className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
 
     case IconName.HISTORY:
-      return <History className={getColor(color)} width={`${size}px`} height={`${size}px`} />;
+      return <History className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
 
     case IconName.HOME:
-      return <Home className={getColor(color)} width={`${size}px`} height={`${size}px`} />;
+      return <Home className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
 
     case IconName.SEARCH:
-      return <Search className={getColor(color)} width={`${size}px`} height={`${size}px`} />;
+      return <Search className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
 
     case IconName.SIGNOUT:
-      return <Signout className={getColor(color)} width={`${size}px`} height={`${size}px`} />;
+      return <Signout className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
 
     case IconName.SMILE:
-      return <Smile className={getColor(color)} width={`${size}px`} height={`${size}px`} />;
+      return <Smile className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
 
     case IconName.WATCH:
-      return <Watch className={getColor(color)} width={`${size}px`} height={`${size}px`} />;
+      return <Watch className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
+
+    case IconName.BURGERMENU:
+      return <BurgerMenu className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
+
+    case IconName.LOGOTIP:
+      return <MainLogo className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
+
+    case IconName.BELL:
+      return <Bell className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
+
+    case IconName.SETTINGS:
+      return <Settings className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
+
+    case IconName.MOON:
+      return <Moon className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
+
+    case IconName.LOGOUT:
+      return <LogOut className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
 
     default:
-      return <Timeago className={getColor(color)} width={`${size}px`} height={`${size}px`} />;
+      return <Timeago className={getColor(color) + ` ${className}`} width={`${width}px`} height={`${height}px`} />;
```

Comment:
1. use clsx for class names https://www.npmjs.com/package/clsx

2. What's the reason to duplicate same props each time?

```javascript
const commonProps: Type = {
  className: ...,
  width: ... // no reason to make it string with px. No need to change but just have in mind
  height: ...,
}
```